ui/src/components/facets/HistogramFacet.js

```diff
@@ -158,6 +160,16 @@ class HistogramFacet extends Component {
             onNewView={this.onNewView}
           />
         </div>
+      );
+    }
+
+    return (
+      <div className={classes.histogramFacet}>
+        <FacetHeader
+          facet={this.props.facet}
+          selectedValues={this.props.selectedValues}
+        />
+        {vegaDiv}
```

Comment:
opt

I find [Inline if](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) easier to read. Then you could keep the vega stuff here. You don't need to change it, just letting you know this is an option for next time.